Converters/ArcGIS/Speckle.Converters.ArcGIS3/Utils/LocalToGlobalConverterUtils.cs

```diff
@@ -28,102 +28,28 @@ public Base TransformObjects(Base atomicObject, List<Matrix4x4> matrix)
       return atomicObject;
     }
 
-    // This is a temp hack. We would add transformations to conversions later instead try to copy objects like this.
-    // Jedd also has opinions on this.
-    Base newObject = Core.Api.Operations.Deserialize(Core.Api.Operations.Serialize(atomicObject));
+    List<Objects.Other.Transform> transforms = new() { };
+    transforms.AddRange(matrix.Select(x => new Objects.Other.Transform(x, "none")).ToList());
```

Comment:
Ideally same as you did but I would simplify it as one liner, and it is better. And also probably we do not necessarily have it as `ToList()` since we won't mutate it later, it is just a mapping one to another.
```c#
IEnumerable<Objects.Other.Transform> transforms = matrix.Select(x => new Objects.Other.Transform(x, "none"));
```